apps/ledger-live-mobile/e2e/models/market/marketPage.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  getElementById,
+  getElementByText,
+  typeTextByElement,
+  tapByText,
+  tapByElement,
+} from "../../helpers";
+
+export default class marketPage {
+  searchBar = () => getElementById("search-box");
+  starBtn = () => getElementById("starAsset");
+  assetCardBtnBack = () => getElementByText("Bitcoin");
+  assetCardBackBtn = () => getElementById("marketBackBtn");
+  starMarketListBtn = () => getElementById("stared");
+  buyNanoBtn = () => getElementById("testIDBuyBtn");
+  openMarketPlaceBtn = () => getElementById("marketPlaceBtn");
+
+  async searchAsset(asset: string) {
+    await typeTextByElement(this.searchBar(), asset);
```

Comment:
For all of these you can replace `await` with `return`. In the test you do an `await` which means you don't need it here. Adding extra unnecessary `awaits` creates unnecessary promises (no worries if you see it in the rest of the code. It's something to tidy up)